File: C/AD-HOC/1397 Jogo do Maior.c
#include<stdio.h>

int main()
{
    int rounds;
    scanf("%d", &rounds);
    while (rounds != 0)
    {
        int p1=0, p2=0;
        while (rounds != 0)
        {
            int n1, n2;
            scanf("%d %d", &n1, &n2);
            if (n1 > n2)
                p1 += 1;
            else if (n2 > n1)
                p2 += 1;    
            rounds -= 1;
        }
        printf("%d %d\n", p1, p2);
        scanf("%d", &rounds);
    }
    return 0;
}

File: Python/AD-HOC/1397 Jogo do Maior.py
# SUBPROGRAMA
def contarPontos(rounds):
    j1 = j2 = 0
    while rounds != 0:
        n1, n2 = map(int, input().split())
        if n1 > n2:
            j1 += 1
        elif n2 > n1:
            j2 += 1
        rounds -= 1
    print(j1, j2)
    return None


# PROGRAMA PRINCIPAL
rodadas = int(input())
while rodadas != 0:
    contarPontos(rodadas)
    rodadas = int(input())
